### DISCUSSION TITLE
How do I disable the scroll programmatically?

### DISCUSSION THREAD

**Discussion by MomoIsHeree** [Norserium/react-indiana-drag-scroll#83] [Q&A]
Hey there,
I've been looking into how to disable the scrolling functionality programmatically and couldnt find anything.
The reason why I need to do it this way, is because I can't use ignoreElements, because the library I'm using (konva.js) just makes a giant canvas element and I haven't found a way to detect if an element inside the canvas is touched.

However, my items have a "dragging" flag, so my current idea is that I just programmatically disable the drag onDragStart, if any elements have the dragging flag set to true.

So, back to the original question: How would I do this programmatically inside onScrollStart()

**Comment by Norserium** [comment]
@MomoIsHeree, what do you mean by disabling the scrolling functionality? Why do you need this library in this case?

**Reply by MomoIsHeree** [reply]
Well, I want to be able to disable the scroll if a certain case sets in. In my example above, I got a canvas, where I drag around items. I want to be able to navigate by dragging as well, but when doing this, I need to be able to check if I'm dragging an item or the canvas and then respectively disable/enable the drag-scrolling functionality.
